server/opts.go

```diff
@@ -627,6 +627,11 @@ type importService struct {
 	to  string
 }
 
+// Checks if an account name is reserved.
+func isReservedAccount(name string) bool {
+	return strings.Compare(name, globalAccountName) == 0
```

Comment:
Why not `return a == globalAccountName`? This is what Compare does.

Reply:
Good point, I wonder if it has always been this way? I will fix.

https://golang.org/src/strings/compare.go?s=490:519#L3